packages/@aws-cdk/aws-ecr/README.md

```diff
@@ -118,3 +118,20 @@ declare const repository: ecr.Repository;
 repository.addLifecycleRule({ tagPrefixList: ['prod'], maxImageCount: 9999 });
 repository.addLifecycleRule({ maxImageAge: Duration.days(30) });
 ```
+
+## Managing the Resource Policy
+
+You can add statements to the resource policy of the repository using the
+`addToResourcePolicy` method. However, be advised that you must not include
+a `resources` section in the `PolicyStatement`. See AWS internal ticket
+`#WRITEME` for details around the ECR and Cfn team working together to
```

Comment:
> See AWS internal ticket `#WRITEME`

Does not seem to fit with the spirit of a public doc. How is a non-Amazon CDK user supposed to get the information?

Reply:
@Naumel we're all Amazon users here. If you mean a non-Amazon employee, they'd ask their TAM. My intention is specifically to have everyone who sees this have their TAM poke the PMs for ECR and Cfn and ask them about this. Speaking of which, I need to bug our TAM about this to get the ticket.

Reply:
@ahammond My opinion: The documentation is technical and fact-based. The purpose is not to put pressure on a team to get a desired feature.

Reply:
We will need this changed before we can merge this change.